specs/engage-digital_openapi3.yaml

```diff
@@ -33,7 +33,7 @@ x-tag-groups:
       - Intervention Comments
       - Interventions
       - Sources
-      - Survey Responses
+      - Surveys

```

Comment:
I Merged survey_response and surveys into the same section, this is debatable as it's not the same url, but it was clearer doc-wise. If you disagree, I can make it into to two separate sections.